src/kiwixchoicebox.cpp

```diff
@@ -112,18 +105,40 @@ KiwixChoiceBox::~KiwixChoiceBox()
     delete ui;
 }
 
-void KiwixChoiceBox::mouseReleaseEvent(QMouseEvent *event)
+void KiwixChoiceBox::mousePressEvent(QMouseEvent *event)
```

Comment:
Almost (all?) toolkit generate a click event on `release` mouse button. Why changing this ?

Reply:
When the `lineEdit` is currently focused,
if the outer widget is *pressed*, `lineEdit` loses focus and hides the options.
If I use `mouseRelease`, it will create a flicker effect: Pressing causes the `lineEdit` to lose focus and hide the options - Release causes the options to show up again.

showing the options on a `mousePress` doesn't allow this

Reply:
Please add this explanation as a comment in the source code.